cli/ballerina-cli/src/test/resources/test-resources/doc_project/Dependencies.toml

```diff
@@ -0,0 +1,18 @@
+# AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
Do we need to commit this Dependencies.toml? If this is not used for any purpose, we can skip adding this. Also if this is used for some assertions, be mindful of the field `distribution-version = "2201.9.0-SNAPSHOT"` since once the distribution version gets changed, this will cause test failures.